### ISSUE TITLE
MNT: Compatibility with data caching in astropy 4.0

### ISSUE THREAD

**Issue by pllim** [astropy/astroquery#1594]
Monkeypatching stuff internal to `astropy.utils.data` functions have broken `astroquery` compatibility with `astropy` 4.0 and later.

See https://github.com/astropy/astroquery/pull/1588#issuecomment-555187571

https://github.com/astropy/astroquery/blob/c96d5f4f306eee44f59de96e77d6f34bc4d784bb/astroquery/utils/tests/test_utils.py#L413

Example CI log: https://travis-ci.org/astropy/astroquery/jobs/613319228

tl;dr -- By setting `_is_url` to `True`, `astropy` now will attempt to download the file from Internet.

**Comment by pllim** [comment]
I don't know the internals of the monkeypatching effort to suggest an effective fix. I see that @keflavich , @cdeil , and @MSeifert04 had touched that piece of code in the past.

For changes on `astropy` side, it started with astropy/astropy#9182

**Comment by pllim** [comment]
Fixed in #1603
